framework/components/ecl-search-forms/README.md

```diff
@@ -2,11 +2,19 @@
 
 ## Why and how to use this component
 
-The search box allow user to search within all EC websites. The search box allows user to search in all European Commission websites through the EC search engine.
+A "Search" functionality helps users to locate content quicker by using specific words or phrases without having to navigate through the entire website.
 
-The search results are displayed in a separate page.
+### Types of search
+
+#### Corporate search
+
+The search box allows the user to search within all EC websites through the EC search engine. The search results are displayed in a separate page.
```

Comment:
Please take also the rest of the information, even if it's not super related to the component ...